src/NUnitFramework/framework/Constraints/Constraint.cs

```diff
@@ -292,15 +292,15 @@ public ConstraintExpression Or
 
 #if !PORTABLE
         /// <summary>
-        /// Returns a DelayedConstraint with the specified delay time.
+        /// Returns a DelayedConstraint.WithRawDelayInterval with the specified delay time.
         /// </summary>
         /// <param name="delayInMilliseconds">The delay in milliseconds.</param>
         /// <returns></returns>
-        public DelayedConstraint After(int delayInMilliseconds)
+        public DelayedConstraint.WithRawDelayInterval After(int delayInMilliseconds)
```

Comment:
This is a bit of a nit pick, but is it possibly confusing to keep the name "delayInMilliseconds" when it may end up being changed? This shows up in intellisense with the explanation "The delay in milliseconds."

Maybe it should be simply "delay" with a description like " The delay, which defaults to milliseconds"